pgmax/fg/graph.py

```diff
@@ -482,24 +394,21 @@ def update_log_potentials(
         (1) Provided log_potentials shape does not match the expected log_potentials shape.
         (2) Provided name is not valid for log_potentials updates.
     """
-    for name in updates:
-        data = updates[name]
-        if name in fg_state.named_factor_groups:
-            factor_group = fg_state.named_factor_groups[name]
-
+    for factor_group, data in updates.items():
```

Comment:
This means there is no way to pass in updates for individual factors. I think this is a case that is currently not covered (solution is to use single factor as keys when adding single factors)

Reply:
Correct, we currently do not provide this option. In factor we do not compute the `factor_to_potentials_starts` indices